docs/dev-overview/get-balance.md

```diff
@@ -1,22 +1,38 @@
-# Get balance
+# Get balances
 
-### User experience
+### Motivation
 
-In order for users to receive transactions (that is, get their balance), they must be able to deposit funds from a [**request QR Code**](../glossary.md).
-
-![Users can get their balance or deposit funds from a transfer QR Code.](../images/get-balance.jpeg)
+To show what a user currently has in their account, MobileCoin wallet applications must be able to
+fetch the account's `Balance`s. A user's account will have a `Balance` for every token/asset which
```

Comment:
Yes, we shouldn't assume that SDK users are strictly creating a wallet application. If we look at Signal, for example, they are using the SDK to add payments to a messaging application. Still though, the class in Signal that utilizes the SDK is named `Wallet`. Because what they're really doing is integrating a MobileCoin wallet into a messaging application. So, in my opinion, we don't necessarily need to avoid saying "wallet application;"  as long as the intended readers/users are developers, I believe they will understand that they can use this SDK to develop applications that are not _strictly_ a MobileCoin wallet.

I think we need to consider this suggestion, and I am interested in hearing input from others on this. I'll think about alternative ways to phrase this section.

Reply:
> 

This is a good suggestion

Reply:
Updated this to be a bit more generic